### PR TITLE
Add an Issue template

### PR DESCRIPTION
**Pull request by AmedeeBulle**
Issue template to capture user environment

Signed-off-by: Philippe Vanhaesendonck <philippe.vanhaesendonck@oracle.com>

### CODE REVIEW

**Comment by AmedeeBulle** [comment]
Tagged as WIP to collect feedback before merging

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
+
+**Environment (please complete the following information):**
+ - Host OS and version: [e.g. Oracle Linux 8.2 UEK6, macOS 10.14.6, Windows 10 Pro, ...]
```
Comment (Djelibeybi) [inline .github/ISSUE_TEMPLATE/issue-report.md:14]: I would separate host OS from kernel version, i.e. have another question, "If you're using Linux, please provide the output from `uname -a`.

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ - Vagrant version:  [e.g. 2.2.9]
+ - Vagrant provider:
+   - For VirtualBox:
+      - VirtualBox version: [e.g. 6.1.8r137981 -- run `vboxmanage -v`]
+   - For libvirt:
+     - QEMU version: [e.g. 3.1.0 -- run `rpm -q qemu-kvm`]
```
Comment (Djelibeybi) [inline .github/ISSUE_TEMPLATE/issue-report.md:20]: Some `libvirt` hosts may not have `rpm`. Would `qemu -v` work?

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+     - Libvirt version: [e.g. 5.7.0 -- run `rpm -q libvirt`]
```
Comment (Djelibeybi) [inline .github/ISSUE_TEMPLATE/issue-report.md:21]: Same here. Is there a non-RPM method to extract the libvirt version?

File: .github/ISSUE_TEMPLATE/issue-report.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+name: Issue report
+about: Create an issue to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue is.
```
Comment (Djelibeybi) [inline .github/ISSUE_TEMPLATE/issue-report.md:11]: Perhaps a more pointed set of questions, like "Explain what commands you ran, what you expected to happen and what actually happened."

File: .github/ISSUE_TEMPLATE/issue-report.md (hunk 2 above)
Comment (AmedeeBulle) [reply .github/ISSUE_TEMPLATE/issue-report.md:20]: Yes I am aware of that -- I'll see if we can have something more generic for these two

Comment (AmedeeBulle) [reply .github/ISSUE_TEMPLATE/issue-report.md:20]: There isn't anything we can rely on cross-distributions.
I have rephrased...

**Pull request merged**